app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TweetController < ApplicationController
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    @tweet = Tweet.find_by_id(params[:id])
+    if logged_in? && current_user.id == @tweet.user.id
+      erb :'tweets/edit_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'tweets/show_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets/new' do
+    if params[:content] == ""
+      redirect "/tweets/new"
+    else
+      @user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.new(content: params[:content],user_id: @user.id)
```

Comment:
use the associative Active Record methods. We should also just go ahead and create his instead of calling save on line 41

`current_user.tweets.create(content: params[:content])`

this will automatically associated the user_id with current_user.id